source/views/student-orgs/list.js

```diff
@@ -178,30 +185,95 @@ export class StudentOrgsView extends React.Component {
     })
   }
 
+  handleResults = (results: {[key: string]: StudentOrgAbridgedType[]}) => {
+    this.setState({data: results})
+  }
+
+  groupData = (
+    data: {[key: string]: StudentOrgAbridgedType[]},
+    groupingArgs: GroupingArgsType,
+  ) => {
```

Comment:
Oh, you're reusing the argument type. I'd prefer to keep them separate, as how to group data feels like an internal detail of the list view that's not inherently sharable between things.